futaba/cogs/settings/core.py

```diff
@@ -342,3 +343,55 @@ def mention(role):
 
         await ctx.send(embed=embed)
         self.journal.send("roles/jail", ctx.guild, content, icon="settings", role=role)
+
+    @commands.command(name="trackblacklistadd")
```

Comment:
There should be at least one alias that shortens this command. Given that this follows the add/remove pattern, I think it's best split out into a command group ("trackerblacklist") with subcommands ("add", "remove", "show"...) (see `!role`)

Reply:
yup, should viewing the blacklist be mod only?

Reply:
I think that makes sense. You still need to apply `@permissions.check_mod()` to each of the subcommands though I believe.

Reply:
You don't if the group command is mod only